service/keys/keys.go

```diff
@@ -51,3 +57,29 @@ func (keys *keys) GetNostrPublicKey() string {
 func (keys *keys) GetNostrSecretKey() string {
 	return keys.nostrSecretKey
 }
+
+func (keys *keys) GetBIP32ChildKey(childIndex uint32) (string, error) {
```

Comment:
Just to confirm, we are ok with the performance impact of this (twice for request/response) and once per notification

Reply:
I'll do some bench-marking just to be sure what is the impact.

Reply:
`GetAppWalletKey average execution time per call (10000 cycles): 2.622242ms`